### PR TITLE
fix server error with dev, minor refactoring

### PR DESCRIPTION
**Pull request by ops1ops**

### CODE REVIEW

**Comment by dexter0175** [comment]
Что за ошибка с сервером была?

**Comment by dexter0175** [comment]
The problem  was with ES6 module imports, is it really necessary to require `dotenv/config` in startup command ?

**Comment by ops1ops** [comment]
> The problem was with ES6 module imports, is it really necessary to require `dotenv/config` in startup command ?

image

**Comment by dexter0175** [comment]
Ok

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "webpack --mode production",
     "start": "npm run build && babel-node src/server/index.js",
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
-    "server": "nodemon src/server/index.js",
+    "server": "nodemon --exec babel-node -r dotenv/config src/server/index.js",
```
[the remark this answers is not in the document]
Comment (ops1ops) [reply package.json]: We need dotenv config for server

**Pull request merged**